dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -41,11 +41,45 @@ const createValidationSchema = (key: string, formSchema: FormPageSchema) => {
   return object({ [key]: validationShape } as ObjectShape);
 };
 
-const createFlatFormSchema = (formSchema: FormPageSchema) => {
-  const newSchema: FormPageSchema = {
+const addChildProperties = (
+  baseFormSchema: FormPageSchema,
+  newFormSchema: FormPageSchema,
+  formValues: FormState,
+) => {
+  let newProperties: Record<string, FormPageSchema> = {};
+  let newRequired: ReadonlyArray<string> = [];
+
+  const childSchema = createFlatFormSchema(newFormSchema, formValues);
+
+  newProperties = {
```

Comment:
Could we just directly assign this to a `const` here instead of defining the `let` on line 49?